cli/src/commands/asset.spec.ts

```diff
@@ -3,10 +3,12 @@ import * as os from 'node:os';
 import * as path from 'node:path';
 import { describe, expect, it, vi } from 'vitest';
 
-import { Action, checkBulkUpload, defaults, Reason } from '@immich/sdk';
+import { Action, checkBulkUpload, defaults, getSupportedMediaTypes, Reason } from '@immich/sdk';
 import createFetchMock from 'vitest-fetch-mock';
 
-import { checkForDuplicates, getAlbumName, uploadFiles, UploadOptionsDto } from './asset';
+import { checkForDuplicates, getAlbumName, startWatch, uploadFiles, UploadOptionsDto } from 'src/commands/asset';
+
+const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
There is a promise based version of setTimeout that with exactly like this. 

https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options

Reply:
TIL, updated